### ISSUE TITLE
Tomahawk + Tint2/openbox GUI issue

### ISSUE THREAD

**Issue by khamryn** [tomahawk-player/tomahawk#483]
Im currently on Gentoo Linux and installed Tomahawk 0.8.4.  Tomahawk seems to works fine, however there seams to be a issues with tint2 or openbox once the application is hidden.  When I left click on Tomahawk Icon on Tint2 system tray, the menubox will hide and I can not longer access it.  Music will continue to play and I still have middle and right click functions, but I cannot unhide it by left click.  I must quit the application ether by kill process or right click -> quit on the icon. 

If testing this is too much for a small demography (openbox/tint2), can I request a unhide option via command-line for future versions or right-click --> unhide from icon.  Thank you.

**Comment by dschmidt** [comment]
Unhide from icon via command line? Doesn't simply running "tomahawk" again from console aquire just this?

**Comment by khamryn** [comment]
No, because Tomahawk is still actually running, its just "hidden", running "tomahawk" again will not unhide it or run another instance.  It will display this message on the console while tomahawk GUI is hidden and icon is in the system tray.  
"Tomahawk is already running, shutting down."

**Comment by dschmidt** [comment]
yeah, but it should bring the running instance to the front. Meh.

**Comment by khamryn** [comment]
Thanks for all work.  I pulled and built from the current git branch.  The hidden icon issue I had experienced is fixed.

**Comment by theli-ua** [comment]
just FYI on gentoo id recommend just unmasking 9999 and emerging that instead of building manually

**Comment by khamryn** [comment]
That's exactly what I did. thank.
